frontend/src/container/ExplorerOptions/ExplorerOptions.tsx

```diff
@@ -564,10 +574,7 @@ function ExplorerOptions({
 							}}
 							dropdownStyle={dropdownStyle}
 							className="views-dropdown"
-							allowClear={{
```

Comment:
removed the clear option from here as we are supporting better / bigger clear selection now